update.php

```diff
@@ -0,0 +1,32 @@
+<?php /*
+*    Pi-hole: A black hole for Internet advertisements
+*    (c) 2017 Pi-hole, LLC (https://pi-hole.net)
+*    Network-wide ad blocking via your own hardware.
+*
+*    This file is copyright under the latest version of the EUPL.
+*    Please see LICENSE file for your rights under this license. */
+    require "scripts/pi-hole/php/header.php";
+?>
+<!-- Title -->
+<div class="page-header">
+    <h1>Update PiHole</h1>
+</div>
+
+<!-- Alerts -->
+<div id="alInfo" class="alert alert-info alert-dismissible fade in" role="alert" hidden>
+    <button type="button" class="close" data-hide="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+    Updating... this may take a while. <strong>Please do not navigate away from or close this page.</strong>
+</div>
+<div id="alSuccess" class="alert alert-success alert-dismissible fade in" role="alert" hidden>
```

Comment:
I see there is an `updating` and `success` section that gets displayed as a valid alert. Do you have any logic that sends back an error message if something fails?

Reply:
This was going to be fixed in a future pull request, initially this is a POC